src/input.js

```diff
@@ -423,12 +423,12 @@ function containerChildrenMutated( mutations ) {
 
 	// Check if all mutations are `children` type, and there is no single text node mutation.
```

Comment:
This comment needs to be synced with the code.

Reply:
And the function description could be too.

Reply:
Function description is fine, but I re-arranged it. The comment will be changed.